### PR TITLE
Set integrated terminal java home and path based on OS

### LINKED ISSUE #44: Installer doesn't properly set java home for mac or linux terminals
Only windows terminals are set currently.

### PR DESCRIPTION
**Pull request by sciencewhiz**
Fixes #44

### CODE REVIEW

File: WPILibInstaller-Avalonia/ViewModels/InstallPageViewModel.cs [hunk 1]
```diff
@@ -267,38 +267,51 @@ private async Task ConfigureVsCodeSettings()
...
             SetIfNotSet("update.showReleaseNotes", false, settingsJson);
 
-            if (!settingsJson.ContainsKey("terminal.integrated.env.windows"))
+            string os;
+            string path_seperator;
+            if  (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                os = "windows";
+                path_seperator = ";";
+            } else if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```
Comment (prateekma) [inline WPILibInstaller-Avalonia/ViewModels/InstallPageViewModel.cs:276]: Should be OSX.

**Review by prateekma — CHANGES_REQUESTED** [review]

**Comment by sciencewhiz** [comment]
It works on windows and linux. On MacOS, the new java path gets appended to the end of the path, rather then the beginning as specified, so it still uses the system java.

**Comment by sciencewhiz** [comment]
https://code.visualstudio.com/docs/editor/integrated-terminal#_why-are-there-duplicate-paths-in-the-terminals-path-environment-variable-andor-why-are-they-reversed

**Comment by sciencewhiz** [comment]
Neither of those two workarounds seem to work on MacOS. I think it's still better with this fix then without (JAVA_HOME is set, and the FRC jdk is added to the path, albeit at the end, so that if there isn't a system java for some reason, it would use the FRC jdk.

**Review by PeterJohnson — APPROVED** [review]

**Pull request merged**